rust-toolchain

```diff
@@ -1 +1 @@
-nightly-2025-01-01
+nightly-2025-02-01
```

Comment:
Rust 1.86 is required

Reply:
This shouldn't be a problem in terms of compatibility with lcp, right?

Reply:
That shouldn't be a problem.
I only need to change the tool chain of enclave.
I have confirmed that it works with the following repositories.
https://github.com/datachainlab/cosmos-optimism-ibc-lcp/blob/feature/kona-client-v1.0.2/rust-toolchain